specification/columns/tags.md

```diff
@@ -25,7 +26,7 @@ The following is an example of one user-defined tag and one provider-defined tag
 ```json
     {
         "foo":"bar",
-        "acme/foo": "bar"
+        "acme/foo":"bar"
```

Comment:
Shall we expand the example to include examples of:
1. labels
2. tags with null values being optionally included

Reply:
+1, would like to see what labels and tags looks like on a single line item as well

Reply:
Example for a label now available